lambda/lambda.mli

```diff
@@ -603,12 +603,40 @@ type let_kind = Strict | Alias | StrictOpt
       we can discard e if x does not appear in e'
  *)
 
+(** A unique barrier is necessary on all references to a value that
```

Comment:
This comment will grow stale quickly, because new primitives and syntax are being added all the time. At the least, it should include a date of validity and some instructions for the analysis you did so that it can be re-created in the future.

But how can it be kept up to date? Maybe leave some breadcrumbs in the right places in uniqueness_analysis.ml so that this is updated whenever a new bit of syntax doesn't count as a unique use?